code/game/objects/items/weapons/material/kitchen.dm

```diff
@@ -90,6 +91,20 @@
 /obj/item/material/kitchen/utensil/spoon/plastic
 	default_material = MATERIAL_PLASTIC
 
+/obj/item/material/kitchen/utensil/chopsticks
+	name = "chopsticks"
+	desc = "A pair of chopsticks. The most challenging utensil in the Spur."
+	icon_state = "chopsticks"
+	default_material = MATERIAL_WOOD
+	transfer_amt = 2 //Chopsticks are hard to grab stuff with
+
+/obj/item/material/kitchen/utensil/chopsticks/New()
```

Comment:
this is inherited from /utensil/New() so no reason to duplicate it

Reply:
Due to me being a complete idiot, this repo was deleted, so I can't make any edits. I will fix this in another PR once this is merged, however.